source/ca.backup2/usr/local/emhttp/plugins/ca.backup2/include/backupExec.php

```diff
@@ -193,12 +193,22 @@ function getDates() {
 
 case 'checkBackup':
   if ( is_file($communityPaths['backupLog']) ) {
-    $backupLines = "<font size='0'><tt>".shell_exec("tail -n10 ".$communityPaths['backupLog'])."</tt></font>";
+    $backupLines = "<font size='0'><tt>".file_get_contents($communityPaths['backupLog'])."</tt></font>";
```

Comment:
This is going to _very quickly_ crash your browser as using Plex as an example, there's hundreds of thousands of logged lines and the browser is going to have to churn through it all in order to render it properly.

Reply:
Did not had any issues so far, because I removed the `v` flag of the tar commands, as they dont improve any error handling, since stderr lines come still up.